metal/loadbalancers/emlb/emlb.go

```diff
@@ -10,24 +10,62 @@ import (
 )
 
 type LB struct {
-	client *lbaas.APIClient
+	client     *lbaas.APIClient
+	controller *controller
 }
 
 func NewLB(k8sclient kubernetes.Interface, config string) *LB {
+	metalAPIKey := "TODO"
```

Comment:
- We can get `metalAPIKey` if we pass in the `Config` object (look for `getMetalConfig` in this repo)
- We can get `projectId` the same way
- We also need the cluster name, but not sure yet where we get that from

Reply:
You'll want to register the implementation here to follow the existing configuration patterns: https://github.com/kubernetes-sigs/cloud-provider-equinix-metal/blob/main/metal/loadbalancers.go#L58-L99

Reply:
The EMLB version of NewLB can differ from the other NewLB function to take whatever configuration arguments are necessary.